docs/upgrade.md

```diff
@@ -0,0 +1,149 @@
+# Upgrading KubeVirt
+
+This document outlines the step by step process of upgrading kubevirt
+from the perspective of the kubevirt-operator.  These steps can also
+be done by hand.
+
+**Major Upgrade** - Upgrade from 1.0 to 2.0
+**Minor Upgrade** - Upgrade from 1.0 to 1.1
+
+### Major Upgrade
```

Comment:
Put the minor upgrade above the major, since major isn't something we're going to handle until a we have a major version to jump to.